SECURITY.md

```
@@ -0,0 +1,53 @@
+| < 1.8   | :x:                |
+
+"Supported" means the version receives security patches. Unsupported versions
+will not receive security fixes.
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability within ACOR, please report it responsibly.
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them privately through [GitHub Security Advisories][security-advisories].
```

markdownlint-cli2 (0.22.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "security-advisories"

(MD052, reference-links-images)